Controllers/BaseController.php

```diff
@@ -38,11 +38,11 @@ public function deleteItem(string $id): void
             }
         }
 
-        Storage::saveElements($filename, $newData);
+        Storage::saveElements($filename, json($newData));
```

Comment:
When writing a code I have found example that used header. Now I understand this is unnecessary because I am not sending json file to client, but I am using it just for encoding on the server.